devices/ds18b20/ds18b20.go

```diff
@@ -175,3 +178,73 @@ func (d *Dev) readScratchpad() ([]byte, error) {
 }
 
 var _ devices.Device = &Dev{}
+
+// W1ThermDevice provides access to DS18B20 devices found on OneWire bus
+type W1ThermDevice struct {
+	Probe map[string]*sysfs.OneWireDevice
+}
+
+// NewW1Therm loads kernel drivers and scans for DSB1820 family devices only
+func NewW1Therm() (*W1ThermDevice, error) {
+	dd := W1ThermDevice{}
+	ow, err := sysfs.NewOneWire()
+	if err != nil {
+		return &dd, err
+	}
+	// Load kernel drivers
+	err = ow.LoadDrivers()
```

Comment:
I'd prefer this function to accept an interface that permits it to scan without depending directly on sysfs. [Bus.Search()](https://godoc.org/periph.io/x/periph/conn/onewire#Bus) currently returns all the addresses found, instead of returning an interesting address, maybe the interface could be adapted to be more polyvalent? Paging @tve.